.github/workflows/main.yml

```diff
@@ -2,20 +2,41 @@ name: Docker image CI
 
 on:
   push:
-    branches: ['master']
-  workflow_dispatch:
+    # takes muliple branch names
+    branches: 
+      - master
+      - '[0-9][0-9][0-9][0-9].[0-9][0-9].*' # 2021.01.01
```

Comment:
This will support build of MyLA release branch if there is [commit](https://github.com/pushyamig/my-learning-analytics/actions/runs/6163072969)

Reply:
Github Packages [outcomes](https://github.com/pushyamig/my-learning-analytics/pkgs/container/my-learning-analytics)

The tagging is like `latest-2023.09.x`